### DISCUSSION TITLE
2.0.0 is out!

### DISCUSSION THREAD

**Discussion by NikiforovAll** [NikiforovAll/keycloak-authorization-services-dotnet#86] [Announcements]
Added new documentation with https://nikiforovall.github.io/keycloak-authorization-services-dotnet/

**Comment by TheAutomaTom** [comment]
Is there any documentation on how to run a web api in docker, connecting to Keycloak in a separate docker container?  

When I try, the exception begins with: 
`Warning: Unable to verify the first certificate`.  

And, at the bottom of the response I see:
`WWW-Authenticate: Bearer error="invalid_token", error_description="The issuer 'http://localhost:8080/realms/my-realm' is invalid"`

This is my config:
```
  "Keycloak": {
    "realm": "my-realm",
    "resource": "my-client",
    //"auth-server-url": "http://localhost:8080/", // for http or https launch profiles
    "auth-server-url": "http://host.docker.internal:8080/",
    "ssl-required": "none",
    "verify-token-audience": false,
    "credentials": {
      "secret": "my-secret"
    }
  },
```
This solution works fine in http and https launch profiles.

I realize this isn't specifically related to your nuget, but I have not found any docs about local development covering this. Thanks!

**Reply by TheAutomaTom** [reply]
Created
https://github.com/NikiforovAll/keycloak-authorization-services-dotnet/issues/115

**Comment by NikiforovAll** [comment]
Hi @TheAutomaTom, it is a common scenario. I will try to reproduce your issue and create an example for it. Could you please create an issue for documentation improvement?
